Earthfile

```diff
@@ -52,84 +52,6 @@ copy-dirs:
-    ENV ROC_NUM_WORKERS=1
-    # run one of the benchmarks to make sure the host is compiled
-    # not pre-compiling the host can cause race conditions
-    RUN gcc --version
-    RUN echo "4" | cargo run --release examples/benchmarks/NQueens.roc
-    RUN --mount=type=cache,target=$SCCACHE_DIR \
-        cargo test --locked --release --features with_sound serde --workspace && sccache --show-stats
-    # test the dev and wasm backend: they require an explicit feature flag.
-    RUN --mount=type=cache,target=$SCCACHE_DIR \
-        cargo test --locked --release --package test_gen --no-default-features --features gen-dev && sccache --show-stats
-    # gen-wasm has some multithreading problems to do with the wasmer runtime. Run it single-threaded as a separate job
-    RUN --mount=type=cache,target=$SCCACHE_DIR \
-        cargo test --locked --release --package test_gen --no-default-features --features gen-wasm -- --test-threads=1 && sccache --show-stats
-    # run `roc test` on Str builtins
-    RUN --mount=type=cache,target=$SCCACHE_DIR \
-        cargo run --release -- test crates/compiler/builtins/roc/Str.roc && sccache --show-stats
-    # repl_test: build the compiler for wasm target, then run the tests on native target
-    RUN --mount=type=cache,target=$SCCACHE_DIR \
-        crates/repl_test/test_wasm.sh && sccache --show-stats
-    # run i386 (32-bit linux) cli tests
-    # NOTE: disabled until zig 0.9
-    # RUN echo "4" | cargo run --locked --release --features="target-x86" -- --target=x86_32 examples/benchmarks/NQueens.roc
-    # RUN --mount=type=cache,target=$SCCACHE_DIR \
-    #    cargo test --locked --release --features with_sound serde --test cli_run i386 --features="i386-cli-run" && sccache --show-stats
-    # make sure website deployment works (that is, make sure build.sh returns status code 0)
-    ENV REPL_DEBUG=1
-    RUN bash www/build.sh
-
-
-verify-no-git-changes:
-    FROM +test-rust
-    # If running tests caused anything to be changed or added (without being
-    # included in a .gitignore somewhere), fail the build!
-    #
-    # How it works: the `git ls-files` command lists all the modified or
-    # uncommitted files in the working tree, the `| grep -E .` command returns a
-    # zero exit code if it listed any files and nonzero otherwise (which is the
-    # opposite of what we want), and the `!` at the start inverts the exit code.
-    RUN ! git ls-files --deleted --modified --others --exclude-standard | grep -E .
-
-test-all:
-    BUILD +test-zig
-    BUILD +test-rust
-    BUILD +verify-no-git-changes
-
-build-nightly-release:
-    FROM +test-rust
-    COPY --dir .git LICENSE LEGAL_DETAILS ci ./
-    # version.txt is used by the CLI: roc --version
-    RUN ./ci/write_version.sh
-    RUN RUSTFLAGS="-C target-cpu=x86-64" cargo build --features with_sound --release
-    RUN ./ci/package_release.sh roc_linux_x86_64.tar.gz
-    SAVE ARTIFACT ./roc_linux_x86_64.tar.gz AS LOCAL roc_linux_x86_64.tar.gz
-
```

Comment:
Most of this has moved to ubuntu_x86_64.yml, spellcheck.yml and nightly_linux_x86_64.yml